src/feelpp/benchmarking/reframe/config/machineConfigs/discoverer.py

```diff
@@ -0,0 +1,50 @@
+site_configuration = {
+    'systems':[
+        {
+            'name': 'discoverer',
+            'descr': 'Discoverer',
+            'hostnames': ['login\d+.discoverer.sofiatech.bg','cn*'],
+            'modules_system': 'tmod4',
+            'partitions': [
+                {
+                    'name': 'cn',
+                    'scheduler': 'slurm',
+                    'launcher': 'srun',
+                    'max_jobs': 8,
+                    'access': ['--partition=cn --account=ehpc-dev-2024d05-047 --qos=ehpc-dev-2024d05-047'],
+                    'environs': ['default'],
+                    'prepare_cmds': [
+                        'source /etc/profile.d/modules.sh',
+                        'export MODULEPATH=/opt/software/modulefiles',
+                        'export OMP_NUM_THREADS=1' # Just in case
```

Comment:
That should not be appear here, it's the application which can define this variables

Reply:
Should we have a benchmark/application configuration field like 
```json 
"env_variables":{ "OMP_NUM_THREADS":1 }
``` 
that will export environment variables at the start of the benchmark?

Reply:
yes

Reply:
Ok, I have added the `env_variables` field under the application config root. 
For containers, they should be passed using their native option under the `options` field. e.g for apptainer: 
```json
 "options": [
   "--env OMP_NUM_THREADS=1"
]
```